packages/atomic/cypress/integration/search-interface.cypress.ts

```diff
@@ -10,38 +12,31 @@ describe('Search Interface Component', () => {
 
   describe('with an automatic search', () => {
     beforeEach(() => {
-      setUpPage('<atomic-query-summary enable-duration>', true);
+      new TestFixture().with(addQuerySummary()).init();
     });
 
     it('should support changing language', () => {
       setLanguage('fr');
-      cy.get('atomic-query-summary')
-        .shadow()
-        .invoke('text')
-        .should('contain', 'Résultats');
+      QuerySummarySelectors.text().should('contain', 'Résultats');
 
       setLanguage('en');
-      cy.get('atomic-query-summary')
-        .shadow()
-        .invoke('text')
-        .should('contain', 'Results');
+      QuerySummarySelectors.text().should('contain', 'Results');
     });
 
     it('should support changing a translation value without overriding other strings', () => {
       setTranslation('fr', 'showing-results-of_plural', 'patate');
       setLanguage('fr');
 
-      cy.get('atomic-query-summary')
-        .shadow()
-        .invoke('text')
-        .should('contain', 'patate')
-        .should('contain', 'seconde');
```

Comment:
Should this assertion be included?

Reply:
I think to respect the intent behind the test, though, we would need to verify that other strings (meaning other than showing-results-of_plural)  haven't been affected